cranelift/codegen/src/machinst/sections.rs

```diff
@@ -352,3 +394,16 @@ pub struct MachCallSite {
     /// The call's opcode.
     pub opcode: Opcode,
 }
+
+/// A source-location mapping resulting from a compilation.
+#[derive(Clone, Debug)]
+pub struct MachSrcLoc {
+    /// The start of the region of code corresponding to a source location.
+    /// This is absolute, not relative to the start of the section.
```

Comment:
Every function is compiled separately without knowledge of the final location in memory, so this can never be absolute. Did you mean relative tot the start of the function?

Reply:
Yup, fixed, thanks!